packages/components/src/Label/src/Label.tsx

```diff
@@ -0,0 +1,68 @@
+import { type StyledComponentProps, useStyledSystem, type ResponsiveProp, useResponsiveValue } from "@hopper-ui/styled-system";
+import { type ForwardedRef, forwardRef, type CSSProperties } from "react";
+import { Label as RACLabel, useContextProps, type LabelProps as RACLabelProps } from "react-aria-components";
+import clsx from "clsx";
+import styles from "./Label.module.css";
+import { cssModule } from "../../utils/src/css-module.ts";
+import { LabelContext } from "./LabelContext.ts";
+
+export const GlobalLabelCssSelector = "hop-Label";
+
+// Won't be needed in next react-aria-components release: https://github.com/adobe/react-spectrum/pull/5850
+const DefaultLabelSlot = "label";
+
+export type RACLabelPropsToOmit = "elementType";
+
+// TODO: Add necessityIndicator and required Props
+export interface LabelProps extends StyledComponentProps<Omit<RACLabelProps, RACLabelPropsToOmit>> {
+    /**
+     * The Typography Type Scale to use.
+     * @default "md"
+     */
+    size?: ResponsiveProp<"inherit" | "xs" | "sm" | "md" | "lg" | "xl" | "2xl">;
```

Comment:
Are there any specs or information in the models about this because I haven't found anything?

Reply:
The usage is documented here:
https://wl-orbiter-website.netlify.app/?path=/docs/responsive-styles--page#responsive-styles

Is this enough?

Reply:
Actually my question is more do we have a reference in a Figma on the different label sizes that are used? Or are these values generic?

Reply:
There is none, i copied the body text sizes. In figma, they use the Text component instead of a Label component, which is ok for them